script.js

```diff
@@ -1,182 +1,153 @@
-// select the <input type="text" class="display" disabled> element
 const display = document.querySelector(".display");
+class Row {
+  constructor(init, name, hp, conditions, hasATurn, id) {
+    this.init = init || 0;
+    this.name = name || "";
+    this.hp = hp || "";
+    this.conditions = conditions || "";
+    this.hasATurn = hasATurn || false;
+    this.id = id || Date.now().toString(36);
+  }
```

Comment:
I feel like it would be nice if you defined all these variables in the class and not just assign them in the init. I'm talking about declaring them like var name; var hp; etc before the constructor, so we know what properties this class has. 

That way, the constructor could be simplified as well, only keeping the properties as parameters that are customisable when creating a row. (Are there any like that? Or is everything defaulted on creation?)